### PR TITLE
Create a response for the Send Item Updates request

### PR DESCRIPTION
**Pull request by jdeeburke**
# Summary
Create a response for the Send Item Updates request

## :vertical_traffic_light: Acceptance criteria

- The new `Facebook\API\Catalog\Send_Item_Update\Response` class is defined.

**Main Story:** [ch54736](https://app.clubhouse.io/skyverge/story/54736)

## QA

- [x] `codecept run integration tests/integration/API/Catalog/Send_Item_Update/ResponseTest.php`

### CODE REVIEW

File: includes/API/Catalog/Send_Item_Update/Response.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ */
+
+namespace SkyVerge\WooCommerce\Facebook\API\Catalog\Send_Item_Update;
+
+defined( 'ABSPATH' ) or exit;
+
+use SkyVerge\WooCommerce\PluginFramework\v5_5_4 as Framework;
+
+/**
+ * Base API response object
```
Comment (wvega) [inline includes/API/Catalog/Send_Item_Update/Response.php:18]:
```suggestion
 * Send_Item_Updates API response object
```

File: includes/API/Response.php [hunk 2]
```diff
@@ -15,7 +15,7 @@
 use SkyVerge\WooCommerce\PluginFramework\v5_5_4 as Framework;
 
 /**
- * Base API response object
+ * Send_Item_Update API response object
```
Comment (wvega) [inline includes/API/Response.php:18]:
```suggestion
 * Base API response object
```

File: includes/API/Catalog/Send_Item_Update/Response.php (hunk 1 above)
Comment (jdeeburke) [reply includes/API/Catalog/Send_Item_Update/Response.php:18]: Hah, good catch. Got my `Resonse` classes mixed up 🤦

**Comment by jdeeburke** [comment]
@wvega renamed and fixed!

File: class-wc-facebookcommerce.php [hunk 3]
```diff
@@ -96,6 +96,8 @@ public function init() {
 				require_once $this->get_framework_path() . '/utilities/class-sv-wp-async-request.php';
 				require_once $this->get_framework_path() . '/utilities/class-sv-wp-background-job-handler.php';
 
+				require_once __DIR__ . '/includes/API/Response.php';
```
Comment (wvega) [inline class-wc-facebookcommerce.php:99]: @jdeeburke I realized we didn't have a story to instantiate the API so I added [one](https://app.clubhouse.io/skyverge/story/55024) and copied Authorize.Net where we require the necessary files in that method right before creating the new instance.

Does that sound good? If it does, I'll make sure move these two to the other method in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1338 now that this one is merged.

Comment (jdeeburke) [reply class-wc-facebookcommerce.php:99]: @wvega Yeah actually I was thinking this felt like the wrong spot for those requires. That's a good move 👍 Thanks!

**Pull request merged**